src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
```

Comment:
You might want to add the javadoc/comment header for the classes and methods! Refer to [this](https://se-education.org/guides/conventions/java/intermediate.html#comments)